<launch>
    <node pkg = "turtlesim" type = "turtlesim_node" name = "turtle1" output="screen" required = "true"/>
    <node pkg = "turtlesim" type = "turtle_teleop_key" name = "key1" output="screen" />
    <node pkg = "turtle_chase" type = "create_turtle" name = "turtle2" output="screen"/>
    <node pkg = "turtle_chase" type = "turtlePose_brocast" name = "turtlePose_brocast1" output="screen" args="turtle1"/>
    <node pkg = "turtle_chase" type = "turtlePose_brocast" name = "turtlePose_brocast2" output="screen" args="turtle2"/>
    <node pkg = "turtle_chase" type = "turtlePose_listen" name = "turtlePose_listen" output="screen"/>
</launch>